composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt

```diff
@@ -26,7 +32,18 @@ data class InstrumentationEntry(
         val stack: String,
         val statusCode: Int,
         val timestampNanos: Long
-)
+) {
+    companion object {
+
+        // See `android.support.test.internal.runner.listener.InstrumentationResultPrinter`.
+
+        const val REPORT_VALUE_RESULT_START = 1
+        const val REPORT_VALUE_RESULT_OK = 0
+        const val REPORT_VALUE_RESULT_FAILURE = -2
+        const val REPORT_VALUE_RESULT_IGNORED = -3
+        const val REPORT_VALUE_RESULT_ASSUMPTION_FAILURE = -4
```

Comment:
I wanted, but then though that there are cases when it's not appropriate, but actually it is, will do